.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: node_js
+cache:
+  directories:
+    - "node_modules"
```

Comment:
can you add the enforcement of the style conventions in the `before_install` section?